__tests__/app.test.js

```diff
@@ -1,15 +1,44 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
-
-/* Set up your beforeEach & afterAll functions here */
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const app = require("../app");
+beforeEach(() => seed(data));
+afterAll(() => db.end());
 
 describe("GET /api", () => {
-  test.skip("200: Responds with an object detailing the documentation for each endpoint", () => {
+  test("200: Responds with an object detailing the documentation for each endpoint", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then(({ body: { endpoints } }) => {
-        expect(endpoints).toEqual(endpointsJson);
+      .then(({ body }) => {
+        expect(body).toEqual(endpointsJson);
+      });
+  });
+});
+
+describe.only("GET /api/topics", () => {
+  test("should return all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
```

Comment:
As we're looping through the response here, it'd be nice to have a length check in an expect statement just before it.